src/networks.json

```diff
@@ -202,5 +202,19 @@
       "https://mainnet-rpc.thundercore.com"
     ],
     "explorer": "https://scan.thundercore.com"
+  },
+  "256": {
+    "key": "256",
+    "name": "Huobi Eco Chain Testnet",
+    "shortName": "heco",
+    "chainId": 256,
+    "network": "testnet",
+    "rpc": [
+      "https://http-testnet.hecochain.com"
```

Comment:
Is this an archive node?

Reply:
sure, try this

curl --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' -H "content-type:application/json" https://http-testnet.hecochain.com